tests/static/all_question_types.xml

```diff
@@ -85,6 +85,7 @@
 					</text>
 					<text id="question1-help">
 						<value>question1 en help</value>
+						<value form="markdown">question1 en help</value>
```

Comment:
I think that would be fine, the phone behavior would be the same either way. The only edge case I can think of is if you were trying to use markdown but the plain text also had markdown in it making it identical so Vellum eats the markdown version even though you wanted that... but even typing that out that just seems completely ridiculous that you'd want plain text with \n and # riddled all over it so, yeah, fine

If the "markdown" form exists it will always be used instead of the normal version FWIW